docs/quickstart.rst

```diff
@@ -98,6 +99,23 @@ But most likely, it will consist in **defining REST resources** using *Cliquet*
 python API !
 
 
+Authentication
+--------------
+
+Currently, if no authentication is set in settings, *Cliquet* relies on
+:term:`Firefox Accounts`, whose setup is tedious.
```

Comment:
I thought this should be "which" but [apparently whose is the right word to use here](http://www.quickanddirtytips.com/education/grammar/whose-for-inanimate-objects?page=all)